cmake/LYTestWrappers.cmake

```diff
@@ -276,7 +276,7 @@ endfunction()
 #
 function(ly_add_pytest)
 
-    if(NOT PAL_TRAIT_TEST_PYTEST_SUPPORTED)
+    if(NOT PAL_TRAIT_TEST_PYTEST_SUPPORTED OR NOT PAL_TRAIT_TEST_LYTESTTOOLS_SUPPORTED)
```

Comment:
I'm fine with this as long as all pytests are failing because of LTT. If there exists any pytests that are currently passing on Linux, then we may need to find something else

Reply:
I'll verify that the test run isn't losing any unnecessary tests.